### ISSUE TITLE
rename module

### ISSUE THREAD

**Issue by alisonjo315** [CU-CommunityApps/CD_cwd_events#27]
Let's make the "human name" of this module more descriptive!

We discussed it extensively over on this [Localist Events Pull issue thread](https://github.com/CU-CommunityApps/CD_cwd_events_localist_pull/issues/18), and noted that this events-embed module name is even vague-r, oof.

I'll share my ideas in the thread.

**Comment by alisonjo315** [comment]
Name/Description suggestions:
(1)
```yml
name: "CWD Localist Events Viewer"
description: "Fetches event data from Localist API (CU calendar, Weill calendar, etc.), and renders fetched data, using a React app, as an embedded object in a Drupal block."
```

(2)
```yml
name: "CWD Localist Embed Block"
description: "same as first idea -- see above"
```

(3)
```yml
name: "CWD Localist Events Embed"
description: "same as first idea -- see above"
```

**Comment by stacypendell** [comment]
The English language must have a perfect verb for what this module does, if we can think what it might be.

"Viewer" doesn't quite work for me because it doesn't make it clear that the events get displayed on the website.

"Embed Block" is good but I'd like to dodge "Block" (because of its multiple meanings, some techy) if we can think of a better word.

"Events Embed" - hmm. The other module (Importer) ended up without "Events" in its name, right? I feel they should have matching names, both either "CWD Localist Events [verb]" or "CWD Localist [verb]". Is it too late to change the other one, because "Localist Events" does seem more descriptive?

Hope I'm not being too confusing by snipping out only the words that are different.

**Comment by alisonjo315** [comment]
> Is it too late to change the other one, because "Localist Events" does seem more descriptive?
NOPE!  I would love for the other module to be called CWD Localist Events Importer!!!!

**Comment by alisonjo315** [comment]
@philwilliammee When you're back, if you have time, let us know what you think! (unless you don't have an opinion about it, which is totally fine -- just didn't want to finalize anything without making sure you're aware of the conversation)
